saleor/graphql/meta/extra_methods.py

```diff
@@ -71,7 +72,10 @@ def extra_transaction_item_actions(instance, info: ResolveInfo, **data):
 
 def extra_user_actions(instance, info: ResolveInfo, **data):
     manager = get_plugin_manager_promise(info.context).get()
-    manager.customer_updated(instance)
+    site = get_site_promise(info.context).get()
```

Comment:
From the help description of the field:

> "When enabled, update webhooks (e.g. `customerUpdated`,"
   "`productVariantUpdated`) are sent even when only metadata changes. "
   "When disabled, update webhooks are not sent for metadata-only changes; "
   "only metadata-specific webhooks (e.g., `customerMetadataUpdated`, "
   "`productVariantMetadataUpdated`) are sent."

It  suggest me that it covers all update-webhooks + update-metadata webhooks. But here, I see only the change for `user` 🤔 It means, that something is wrong or missing.

If this new flag is generic, then we need to also cover cases like the above `(extra_product_actions, extra_variant_actions). 
If this new flag is only related to the customer-metadata, then we need to change the name to be event specific.